frontend/packages/operator-lifecycle-manager/src/plugin.tsx

```diff
@@ -184,19 +184,6 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).CreateSubscriptionYAML,
     },
   },
-  {
-    type: 'Page/Route',
-    properties: {
-      exact: true,
-      path: `/k8s/ns/:ns/${models.ClusterServiceVersionModel.plural}/:appName/:plural/~new`,
-      loader: async () =>
-        (
-          await import(
-            './components/operand/create-operand' /* webpackChunkName: "create-operand" */
-          )
-        ).CreateOperandPage,
-    },
-  },
```

Comment:
I don't think you can remove this route as it seems there are other references in the code.

Reply:
This is a duplicate route.

Reply:
Ah i see where the other is now :)